script/Arcueid/Day 1/01_00_ARC01_2.txt

```diff
@@ -647,12 +645,12 @@ Arihiko also lives near the north gate, so if he's avoiding these 'new idiots',
 [sha:404bd0dd930ad310f077a3852433d54abd8965d9]{
 -- Page 116, Offset 1462.
 -- 「遠野も聞いた事ねえか？
-"Haven't you heard of them?
+"You've heard about it right?
 }
 [sha:9e511dc4433891dd6ac8f5bef9d8e4cb33e62fc3]{
 -- Page 116, Offset 1463.
 -- “度胸試しのようなもの。秘密の入り口から地下道に入って、奥まで着ければ大金が手に入る”ってヤツ」
-It's like a test of courage. You go into an underground passage through a secret entrance, and if you make it to the end, you win a lot of money, or so they say."
+The thing that sounds kinda like a test of courage. Go into an underground passage through a secret entrance, and if you make it to the end, you win a lot of money."
```

Comment:
Tried to just connect it better with the line before. I think it flows more logically this way.

Reply:
Kinda prefer original as well, partly because I'm not sure where the positive form (you've heard about it, right?) is coming from on the line above? I would have thought 遠野も聞いた事ねえか would be "You haven't heard about it either?"

Reply:
That's probably true, not that it makes much difference: 遠野も is the form used so the inclusiveness there made me think it was more, 'c'mon you've heard this one' or something like that.